pyproject.toml

```diff
@@ -0,0 +1,49 @@
+
+[project]
+name = "safeeyes"
+version = "2.2.2"
+description = "Protect your eyes from eye strain using this continuous breaks reminder."
+keywords = ["linux utility health eye-strain safe-eyes"]
+readme = "README.md"
+authors = [
+    {name = "Gobinath Loganathan", email = "slgobinath@gmail.com"},
+]
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "Environment :: X11 Applications :: GTK",
+    "Environment :: Other Environment",
+    "Intended Audience :: End Users/Desktop",
+    "License :: OSI Approved :: GNU General Public License v3 (GPLv3)",
+    "Operating System :: POSIX :: Linux",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+    "Topic :: Utilities",
+]
+dependencies = [
+    "PyGObject",
```

Comment:
Please pin dependencies version. Probably, the versions will be different from the ones used in latest release but anyway, we should start from some versions

Reply:
Oh, sorry, forgot that `poetry` or some dependency management tool is not used, comment above is irrelevant